plugins/UsersManager/Model.php

```diff
@@ -241,10 +242,17 @@ public function getUsersHavingSuperUserAccess()
     public function updateUser($userLogin, $password, $email, $alias, $tokenAuth)
     {
         $this->updateUserFields($userLogin, array(
-             'password'   => $password,
-             'alias'      => $alias,
-             'email'      => $email,
-             'token_auth' => $tokenAuth
+            'password'   => password_hash($password, PASSWORD_BCRYPT),
```

Comment:
We have `password_hash(..., PASSWORD_BCRYPT)` quite often. Can we maybe put it in a static method somewhere or does it maybe rather not make sense? Like `Auth::hashPassword($password)`. Maybe same for other rehash methods etc?